buildspec-release.yml

```diff
@@ -22,15 +19,3 @@ phases:
 
       # run a subset of the integration tests
       - IGNORE_COVERAGE=- tox -e py36 -- tests/integ -m canary_quick -n 64 --boxed --reruns 2
-
-      # generate the distribution package
-      - python3 setup.py sdist
-
-      # publish the release to github
-      - git-release --publish
-
-artifacts:
```

Comment:
was this being used as a form of archiving?

Reply:
No, it was just stored as an artifact for the purpose of passing it to the next stage.